docs/user-guide/calendar/README.md

```diff
@@ -2,84 +2,124 @@
+![Calendar overview](calendar-overview.png)
 
-The current month is automatically selected. You can change the  date range between month or week and navigate through the views with the arrows on the left. The **today** button brings you back to the current date.
+- You can create a new calendar by clicking on either the **+ Calendar** button at the top right corner near your avatar or at the bottom of the sidebar on the left.
+  
+- Click on an existing (saved) calendar to view it.
+  
+- You can change the visibility settings of any calendar by clicking on **[⋮] (more)** -> **Visibility settings.**
+  
+    - Any calendar that has the **Favored** option checked will be displayed under the **Favorite** heading in the sidebar to the left.
+      
+    - Any calendar that has the **Public** option checked will be visible to all members of the current project and listed under the **Public** heading in the sidebar to the left.
+      
+    - Any calendar that has the **Public** option unchecked will be considered **Private** and displayed under the **Private Views** heading in the menu bar to the left.
+      
 
-Hovering over one item displays more information to a work package, such as the type, status, start and finish date, assignee and priority, as well as the project to which the work package belongs.
+## Using the calendar
 
-Click on any item to open the work package details or to edit an item.
+### Viewing work packages
 
-![Calendar overview](calendar-overview.png)
+![Calendar module month view](calendar-month.png)
+
+When you open a new calendar, all work packages within the current project are displayed as horizontal strips that span all dates between the start and finish dates (inclusive). The color represents the [work package type](../../system-admin-guide/manage-work-packages/work-package-types). The subject or title displayed on the strip (insomuch as the length allows).
+
+The current month is automatically selected. 
+
+![Calendar time controls](Calendar-timeControls.png)
 
-The numbers in the instruction below correspond to the numbers in the image above:
+1. You can use the previews/next arrow controls (←, →) in the top left corner to move forwards or backwards in time. 
+2. The **today** button brings you back to the current date.
+3. You can use the month/week toggle on the top right corner to switch between either a month view or a week view. 
 
-1. Edit/change the name of your calendar or add additional calendars. Each calendar can be customized to display work packages from specific project(s). You can further fine-tune the information displayed using the filters.
+When multiple work packages are scheduled or span a single day, they are displayed in a vertical stack.
```

Comment:
@psatyal do you mean to say "if multiple work packages are scheduled during the same day" ?

Reply:
Indeed. Your phrasing is clearer, let's use that one 👍